src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/Constants.java

```diff
@@ -2,15 +2,29 @@
 
```

Comment:
I was wondering if it would be a good idea to have constants class for individual report to be clear that total_within_4weeks is for hearings to judgment report for example

Reply:
I've moved the not shared constants into their respective report data classes where they're used